hotel/src/usb/constants.rs

```diff
@@ -14,7 +14,141 @@ const MAX_NORMAL_ENDPOINTS: u16 = 16;
 pub const MAX_PACKET_SIZE: u16 = 64;
 // const FIFO_RAM_DEPTH: u16        = 1024;
 
+// Ask Amit 
```

Comment:
?

Reply:
I trust that equation is correct, but didn't totally understand all of its terms.